CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13.4)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
@chir-set The CMake minimum for 3D Slicer is 3.16.3. Where 3.13.4 was actually an old minimum for 3D Slicer. It was updated in https://github.com/Slicer/Slicer/commit/664fecaf8d70acdfaba30a473fe35931a4999e47. You will want to maintain this CMake minimum policy like Slicer.

Reply:
It's updated in #146 that I plan to merge after some more testing.

Thank you.